bundles/EcommerceFrameworkBundle/Command/IndexService/BootstrapCommand.php

```diff
@@ -47,6 +48,16 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $createOrUpdateIndexStructure = $input->getOption('create-or-update-index-structure');
         $objectListClass = $input->getOption('object-list-class');
         $listCondition = $input->getOption('list-condition');
+        $tenants = count($input->getOption('tenant')) ? $input->getOption('tenant') : null;
+
+        //set active tenant workers.
+        if (!empty($tenants)) {
+            $tenantWorkerList = [];
+            foreach ($tenants as $tenantName) {
+                $tenantWorkerList[] = Factory::getInstance()->getIndexService()->getTenantWorker($tenantName);
+            }
+            Factory::getInstance()->getIndexService()->setTenantWorkers($tenantWorkerList);
```

Comment:
there is no setter `setTenantWorkers` in indexservice.

I'm also not sure, if we should just add the setter and modify the total tenant list due to potential side effects.
Maybe it would be better to refactor the `IndexUpdater`, remove the static methods, make it a service and add the possibility there to inject a customer index service instead of using the global one.

Reply:
Sorry, I forgot to push the part of the IndexService, where the workers are set.

Reply:
I just added the setter to be complete.